### ISSUE TITLE
[Windows] Examples fail with 0xc0000139 STATUS_ENTRYPOINT_NOT_FOUND

### ISSUE THREAD

**Issue by KongouDesu** [rust-windowing/glutin#1269]
I'm following the instructions in the readme:
```
git clone https://github.com/rust-windowing/glutin
cd glutin
cargo run --example window
```

The example compiles without error, however the program immediately exits with the error:
```
Finished dev [unoptimized + debuginfo] target(s) in 32.54s
     Running `target\debug\examples\window.exe`
error: process didn't exit successfully: `target\debug\examples\window.exe` (exit code: 0xc0000139, STATUS_ENTRYPOINT_NOT_FOUND)
```

A similar story goes for the rest of the examples. 

The error code seems to typically occur as a result of missing or damaged dependencies. `winit` seems to compile and run it's examples just fine, so I'm guessing the issue lies somewhere in `glutin`. Perhaps I'm missing some dependencies?

I'm not quite sure exactly what to make of this error. I'm assuming that it's **my setup** that's misconfigured, but not quite sure how. What dependencies could be missing? Is there some prerequisites I'm supposed to install first?

I'd appreciate any hints as to what could be wrong.

Configuration:
OS: `Windows 7, 64-bit`
GPU: `NVIDIA GeForce GTX 1070`
GPU Driver ver.: `441.87 (01/06/2020)`
rustc: `rustc 1.41.0 (5e1a79984 2020-01-27)`
Toolchain: `stable-x86_64-pc-windows-gnu (default)`

**Comment by goddessfreya** [comment]
cc @Osspial

**Comment by KongouDesu** [comment]
The issue still stands using `stable-x86_64-pc-windows-msvc`
I did a full reinstall of MSYS2, currently with `gcc --version: 9.2.0` and `ld --verision: 2.33.1` to no avail.

I tried to run an example from [ggez](https://github.com/ggez/ggez), which uses `glutin`, which **DOES** compile and run. I noticed it depends on `glutin` verson `0.20.1`. Retrying the `glutin` example at the `0.20.0` release **compiles and runs without error**.

I then also tried for `glutin` version `0.21.0`. I had to turn off `deny(warnings)` to suppress some deprecation warnings, but other than that it **builds and compiles without error**.

Next obvious step: try `0.22.0`. This one _does not run_, producing the originally mentioned error.

In conclusion, the error was introduced somewhere between the `0.21.0` and `0.22.0` release. The examples don't seem to have been updated since `0.20.0`. Perhaps they are out of date?

If needed, I can go find the exact commit where it stops working.

**Comment by KongouDesu** [comment]
This issue seems to have been fixed with #1275 by updating `winit`. `winit` issue seemingly fixed in [winit#1386](https://github.com/rust-windowing/winit/issues/1386)

Issue is gone in `winit 0.21` and in turn in `glutin 0.23`

Closing.
